src/scripts.js

```diff
@@ -68,37 +65,45 @@ const userForms = document.querySelectorAll('.user-form');
+const activityFormMinutes = document.querySelector('#activityFormMinutes')
+const activityFormStairs = document.querySelector('#activityFormStairs')
+const hydrationFormOunces = document.querySelector('#hydrationFormOunces')
+const sleepFormHours = document.querySelector('#sleepFormHours')
+const sleepFormQuality = document.querySelector('#sleepFormQuality')
 
+let activityRepo, randomUser, userRepo, sleepRepo, hydrationRepo; // may not need userRepo globally
 
 function loadThisFirst() {
   Promise.all([sleepDataAPI, userDataAPI, activityDataAPI, hydrationDataAPI])
     .then((values) => {
-    displayInfo(values[1].userData, values[3].hydrationData, values[0].sleepData)
-    // let sleepRepo = new Sleep(values[0].sleepData);
-    let activityRepo = new Activity(values[2].activityData);
-    // let today = makeToday(userRepo, userNowId, values[3].hydrationData);
-    // let randomHistory = makeRandomDate(userRepo, userNowId, hydrationData);
-    // historicalWeek.forEach(instance => instance.insertAdjacentHTML('afterBegin', `Week of ${randomHistory}`));
-    // addHydrationInfo(userNowId, hydrationRepo, today, userRepo, randomHistory);
-    // addSleepInfo(userNowId, sleepRepo, today, userRepo, randomHistory);
-    // let winnerNow = makeWinnerID(activityRepo, userNow, today, userRepo);
-    // addActivityInfo(userNowId, activityRepo, today, userRepo, randomHistory, userNow, winnerNow);
-    // addFriendGameInfo(userNowId, activityRepo, userRepo, today, randomHistory, userNow);
-  })
+      // let sleepRepo = new Sleep(values[0].sleepData);
+      activityRepo = new Activity(values[2].activityData);
+      // let today = makeToday(userRepo, userNowId, values[3].hydrationData);
+      // let randomHistory = makeRandomDate(userRepo, userNowId, hydrationData);
+      // historicalWeek.forEach(instance => instance.insertAdjacentHTML('afterBegin', `Week of ${randomHistory}`));
+      // addHydrationInfo(userNowId, hydrationRepo, today, userRepo, randomHistory);
+      // addSleepInfo(userNowId, sleepRepo, today, userRepo, randomHistory);
+      // let winnerNow = makeWinnerID(activityRepo, userNow, today, userRepo);
+      // addActivityInfo(userNowId, activityRepo, today, userRepo, randomHistory, userNow, winnerNow);
+      // addFriendGameInfo(userNowId, activityRepo, userRepo, today, randomHistory, userNow);
+      displayInfo(values[1].userData, values[3].hydrationData, values[0].sleepData)
+    })
 }
 
 function displayInfo(userInfo, hydrationInfo, sleepInfo) {
-  let userRepo = new UserRepo(userInfo);
-  let randomUser = pickUser(userRepo);
+  userRepo = new UserRepo(userInfo);
+  randomUser = pickUser(userRepo);
 
   addInfoToSidebar(randomUser, userRepo);
   displayHydrationInfo(userRepo, randomUser.id, hydrationInfo);
   displaySleepInfo(userRepo, randomUser.id, sleepInfo);
 }
 
 function displayHydrationInfo(userInfo, id, hydrationInfo) {
-  let hydrationRepo = new Hydration(hydrationInfo);
```

Comment:
I think you accidentally deleted the wrong thing when merging here. I removed `makeToday()` in my last PR, as there was a redundant method in UserRepo called `.getToday()`.

Reply:
fixed and pushed up